cmd/create_data_stream.go

```diff
@@ -10,13 +10,12 @@ import (
 	"maps"
 	"slices"
 
-	"github.com/AlecAivazis/survey/v2"
-
 	"github.com/spf13/cobra"
 
 	"github.com/elastic/elastic-package/internal/packages"
 	"github.com/elastic/elastic-package/internal/packages/archetype"
 	"github.com/elastic/elastic-package/internal/surveyext"
+	"github.com/elastic/elastic-package/internal/tui"
```

Comment:
Nit. Given that this is an internal package we could even keep it calling `survey` :slightly_smiling_face:

Reply:
I think it makes sense to leave it as tui, since it could do more than just ask survey questions in the future